pkg/apis/pingcap/v1alpha1/tidbcluster.go

```diff
@@ -299,11 +299,7 @@ func (tc *TidbCluster) GetClusterID() string {
 }
 
 func (tc *TidbCluster) IsTLSClusterEnabled() bool {
-	enableTLCluster := tc.Spec.EnableTLSCluster
-	if enableTLCluster == nil {
-		return defaultEnableTLSCluster
-	}
-	return *enableTLCluster
+	return tc.Spec.TLSCluster != nil && tc.Spec.TLSCluster.Enabled
```

Comment:
It seems we are no longer using the `tc.Spec.EnableTLSCluster`, do we need to remove this property or set it as deprecated?

Reply:
removed